easybuild/easyconfigs/b/Boost/Boost-1.81.0-intel-compilers-2022.2.1.eb

```
@@ -0,0 +1,35 @@
+##
+# Authors::   Denis Kristak <thenis@inuits.eu>
+##
+name = 'Boost'
+version = '1.81.0'
+
+homepage = 'https://www.boost.org/'
+description = """Boost provides free peer-reviewed portable C++ source libraries."""
+
+toolchain = {'name': 'intel-compilers', 'version': '2022.2.1'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://boostorg.jfrog.io/artifactory/main/release/%(version)s/source/']
+sources = ['%%(namelower)s_%s.tar.gz' % '_'.join(version.split('.'))]
+patches = ['%(namelower)s-%(version)s_intel.patch']
```

Could you make this filename a bit more descriptive, and use an uppercase B in the filename?

I had a quick look at the Intel page, perhaps something like one of the following?
```suggestion
patches = ['Boost-%(version)s_icpx_emit_pch.patch']
```
or
```suggestion
patches = ['Boost-%(version)s_icpx_fix_unsupported_argument.patch']
```

---

Done as suggested - thanks! :-)